[android] Torchscript print to logcat

IvanKobzarev opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#31456 [android] Torchscript print to logcat**

External request https://discuss.pytorch.org/t/jit-android-debugging-the-model/63950

By default torchscript print function goes to stdout. For android it is not seen in logcat by default. 
This change propagates it to logcat.

Differential Revision: [D19171405](https://our.internmc.facebook.com/intern/diff/D19171405)

IvanKobzarev committed:
[android] Torchscript print to logcat

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 29593bf:
* **1/1** broken upstream at merge base 148bcd3 (see [grid view](https://dr.pytorch.org/master-timeline.html?count=1&sha1=148bcd3ee50ca547288da64d7e3ecb65178e3b40&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35))
  <details>
  <summary>
  You may want to rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  
* **0/1** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=29593bf99bbd0840a559f1289355cb521283305c).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably caused by upstream breakages:
* [pytorch_linux_xenial_py3_6_gcc5_4_test](https://dr.pytorch.org/build-details.html?build_id=77166721)



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

IvanKobzarev committed:
Update on "[android] Torchscript print to logcat"


External request https://discuss.pytorch.org/t/jit-android-debugging-the-model/63950

By default torchscript print function goes to stdout. For android it is not seen in logcat by default. 
This change propagates it to logcat.

Differential Revision: [D19171405](https://our.internmc.facebook.com/intern/diff/D19171405)

IvanKobzarev committed:
Update on "[android] Torchscript print to logcat"


External request https://discuss.pytorch.org/t/jit-android-debugging-the-model/63950

By default torchscript print function goes to stdout. For android it is not seen in logcat by default. 
This change propagates it to logcat.

Differential Revision: [D19171405](https://our.internmc.facebook.com/intern/diff/D19171405)

[ghstack-poisoned]
